_attributes/common-attributes.adoc

```diff
@@ -63,6 +63,8 @@
 :JaegerName: Red Hat OpenShift distributed tracing platform (Jaeger)
 :JaegerShortName: distributed tracing platform (Jaeger)
 :JaegerVersion: 1.47.0
+//Kubernetes
```

Comment:
As per

![image](https://github.com/user-attachments/assets/846829de-1869-4d7d-aba2-04f1ee3f33cb)

Might be worth asking a DPM or posting in the team-ocp-docs channel. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#attribute-files

Reply:
@dfitzmau I appreciate your diligence. The Service Mesh 3.0 project uses the standalone doc model. By implication, this approach requires a separate attributes file. In the description I noted that this is a TP release, and that I have worked with Kathryn.  I know the standalone doc model can be confusing because it's different. I'd be happy to chat with you if you have questions.

Reply:
Thanks, Tim. All is good in that regard. I missed the description comment. My apologies.

Reply:
No need to apologize. Like I said, I know that the layered doc approach is different, so it's only natural to have some questions.